### PR TITLE
Fix model menu 9x

### PR DESCRIPTION
**Pull request by dgatf**
This PR fixes broken model menu for SKY9X and AR9X radios

Before:
captura_ar9x_21-02-08_16-14-20

After:
captura_ar9x_21-02-08_16-21-44

### CODE REVIEW

File: radio/src/gui/128x64/model_setup.cpp [hunk 1]
```diff
@@ -443,30 +443,15 @@ void menuModelSetup(event_t event)
...
-      MULTIMODULE_MODULE_ROWS(EXTERNAL_MODULE)
-      FAILSAFE_ROWS(EXTERNAL_MODULE),
-      IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 1),                     // Range check and Register buttons
-      IF_PXX2_MODULE(EXTERNAL_MODULE, 0),                          // Module options
-      IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),                     // Receiver 1
-      IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),                     // Receiver 2
-      IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),                     // Receiver 3
-
+    EXTERNAL_MODULE_ROWS
+#if defined(PCBSKY9X) && defined(REVX)
```
Comment (3djc) [inline radio/src/gui/128x64/model_setup.cpp]: #if define are not allowed in menu structure, it breaks compilation under windows

Comment (dgatf) [reply radio/src/gui/128x64/model_setup.cpp]: It sounds like I heard this issue before... may be you mention in another PR time ago. Thanks. Fixed

**Comment by dgatf** [comment]
Last commit update:

- Output type (opendrain/pushpull) is only applicable for 9xrpro with ppm module. Fixed
- Menu refactor to combine taranis and 9x radios

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**